index.html

```diff
@@ -0,0 +1,174 @@
+            <h2 class="content-header">Multi-Post <br>
+              Stories</h2>
+            <ul class="specification">
+              <li class="name">Aliza</li>
+              <li class="skills"><img src="./images/dot.png" alt="">
+              <p> Back End Dev</p></li> 
+              <li class="year"><img src="./images/dot.png" alt="">
+                <p>2015</p></li>
+            </ul>
+            <p class="content-para">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Alias consectetur nobis quod tempora, ad illum!</p>
+            <ul class="technologies">
+                <li class="html technologies1">Html</li>
+                <li class="Css technologies1">Css</li>
+                <li class="javascript technologies1">Javascript</li>
+            </ul>
+            <button class="see-project">See project</button>
+        </div>
+    </div>
+    <div class="container">
+        <div class="image">
+            <img class="img" src="./images/tonic_2.png" alt="">
+        </div>
+        <div class="content">
+            <h2 class="content-header">Tonic</h2>
+            <ul class="specification">
+              <li class="name">Aliza</li>
+              <li class="skills"><img src="./images/dot.png" alt="">
+              <p> Back End Dev</p></li> 
+              <li class="year"><img src="./images/dot.png" alt="">
+                <p>2015</p></li>
+            </ul>
+            <p class="content-para">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Alias consectetur nobis quod tempora, ad illum!</p>
+            <ul class="technologies">
+                <li class="html technologies1">Html</li>
+                <li class="Css technologies1">Css</li>
+                <li class="javascript technologies1">Javascript</li>
+            </ul>
+            <button class="see-project">See project</button>
+        </div>
+    </div>
+    <div class="container">
+        <div class="image">
+            <img class="img" src="./images/multi_2.png" alt="">
+        </div>
+        <div class="content">
+            <h2 class="content-header">Multi-Post <br>
+              Stories</h2>
+            <ul class="specification">
+              <li class="name">Aliza</li>
+              <li class="skills"><img src="./images/dot.png" alt="">
+              <p> Back End Dev</p></li> 
+              <li class="year"><img src="./images/dot.png" alt="">
+                <p>2015</p></li>
+            </ul>
+            <p class="content-para">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Alias consectetur nobis quod tempora, ad illum!</p>
+            <ul class="technologies">
+                <li class="html technologies1">Html</li>
+                <li class="Css technologies1">Css</li>
+                <li class="javascript technologies1">Javascript</li>
+            </ul>
+            <button class="see-project">See project</button>
+        </div>
+    </div>
+  </section>
+<section class="about-container">
```

Comment:
Please consider making about-container `display: flex` in the styles as asked by the previous code reviewer to meet the project requirements. 

## MORE ELABORATION:-

You just need to make them display flex and that's it:-

```
.about-container{
  display: flex;
  flex-direction: column;
}
```